### ISSUE TITLE
add possibility to generate the xml report

### ISSUE THREAD

**Issue by sullenor** [sullenor/wdio-teamcity-reporter#3]

**Comment by HairyMaclary** [comment]
Yes. What does this module actually produce? I can't see any artifacts in TeamCity.

**Comment by kisenka** [comment]
It's produce only proper output to tell TeamCity about tests. But it doesn't produce any files output, so task it's about.

**Comment by HairyMaclary** [comment]
Thanks for the reply. Could you be a little more specific about what is produced? According to TeamCity I actually need an artifact:

https://confluence.jetbrains.com/display/TCD10/Including+Third-Party+Reports+in+the+Build+Results

And when I use a similar package for Jest (with the similar set up instructions) TeamCity automatically produces a new test tab and tells me how many tests have passed in the overview.

**Comment by kisenka** [comment]
It's strange because I use this reporter and get following:
<img src='https://shipusercontent.com/cc239e23cbde01d122f7b8dd423b23bc/Pasted%20Image%201.png' title='Pasted Image 1.png' width=574>

**Comment by kisenka** [comment]
@HairyMaclary which TeamCity version is used?

**Comment by kisenka** [comment]
@HairyMaclary ping

**Comment by HairyMaclary** [comment]
Hi Sorry about the delay. The TC version is 10.0.4. But I think my problem has been a conflict with the Jest TC reporter. My webdriver tests are actually Spectron tests that are run within Jest. So a big confusing mess. Please don't leave this issue open because of me.
